### ISSUE TITLE
[Feature Request]: Checkout to Department

### ISSUE THREAD

**Issue by najirod** [grokability/snipe-it#12995]
### Is your feature request related to a problem? Please describe.

-

### Describe the solution you'd like

I think it would be great for to Departments act as "Users"... I would like to checkout assets and other to departments (IT, DEV, ACC, Marketing ..) Implementing a user-like view would enable easy tracking of equipment and license costs, as well as identifying responsible managers.

**Comment by snipe** [comment]
It's very unlikely that we'll build this, if I'm honest. It's already complicated enough checking things out to assets, locations, users, etc - and a Department cannot be held responsible if something comes back broken or missing.

**Comment by Black-Pawn-C7** [comment]
We also have a use case for this at the moment the manager holds all the assets and its a pain if he quits or if there is a new deparment head.

**Comment by snipe** [comment]
> We also have a use case for this at the moment the manager holds all the assets and its a pain if he quits or if there is a new deparment head.

That's exactly why you wouldn't want to do it that way though. What if he quits and walks off with assets? It is, quite literally, the point of the software to have to check it back in when someone leaves.

**Comment by Black-Pawn-C7** [comment]
I agree with you to a certain point. However that would also apply to book to a location which is supported by snipeIT in a sense this is the same thing with Departments. If the Asset does not show up at audit the department has to explain where it is and has to then replace it.

**Comment by snipe** [comment]
> However that would also apply to book to a location which is supported by snipeIT in a sense this is the same thing with Departments

Yep, and we hate it and fought against it for years for the same reason. Making a bad decision worse by doubling-down doesn't seem like a wise move on our part, IMHO.

**Comment by BraindamageBE** [comment]
Here in our company we have notebooks assigned to people and only used by a single person. But in the warehouse there are desktops used by multiple people. 
We also have "flexible" workplaces with monitors and docking stations in different departments where people can work. Those are also not assigned to people.
I understand your reasoning and you are building the app, so It's offcourse up to you :) but there are cases where assigning it to departments is the logical thing to do, to better keep track of the assets and there location within a "location".
Thanks

Also thanks for this great tool!

**Comment by apitamy** [comment]
In an emergency room where multiple nurses are using the same asset 24/7, how does SnipeIT want me to handle this?

**Comment by snipe** [comment]
@BraindamageBE - couldn't you just use child locations for that?

**Comment by BraindamageBE** [comment]
> [@BraindamageBE](https://github.com/BraindamageBE) - couldn't you just use child locations for that?

Thanks for your reply.

How can you use child locations?
We have our company with locations in a few countries. I didn’t see things about child locations that can be assigned to normal locations.

Thanks

**Comment by snipe** [comment]
I don't see this ask on the short term roadmap, so I'm going to close this for now. 

To create child locations, just pick a different location as the parent.
